doc/releases/release-notes-3.7.rst

```diff
@@ -269,6 +269,41 @@ Drivers and Sensors
 
 * ADC
 
+  * Added ADC_DT_SPEC_*BY_NAME() macros to get ADC io-channel information from DT by name.
+  * Added support for voltage biasing:
+    * Added a :kconfig:option:`CONFIG_ADC_CONFIGURABLE_VBIAS_PIN` selected by drivers that support
+      voltage biasing.
+    * Added a `zephyr,vbias-pins` to the adc-controller base binding to describe voltage
+      bias pins.
+    * Implemented for the ADC114s08 ADC driver.
```

Comment:
```suggestion
    * Implemented TI ADC114s08 ADC driver.
```
better?

Reply:
I can add TI but this is supposed to be a sub bullet of the voltage biasing addition, I am not sure if I'm doing it right